example/lib/login_page.dart

```diff
@@ -40,8 +40,9 @@ class LoginPage extends StatelessWidget {
                         children: [
                           ElevatedButton(
                               child: const Text("Login"),
-                              onPressed: () {
-                                frontegg.login();
+                              onPressed: () async {
+                                await frontegg.login();
+                                print("Login Finished");
```

Comment:
build check failed because of print - we can add check for env, use debugPrint or other package for avoiding it

Reply:
Done